exercises/sheet4/sheet4.tex

```diff
@@ -74,11 +110,79 @@
   \]
   for any polytope $P$, where $p_0 = \frac{1}{\vol(P)}\int_P x\,\text{d}x$ denotes the barycenter of $P$.
   \bigskip\bigskip
-\item Complete the proof of Theorem 9.6 in Ziegler's \emph{Lectures on Polytopes}, possibly referring to \cite{bs-1992}.
+\item Complete the proof of Theorem 9.6 in Ziegler's \emph{Lectures on Polytopes}.
   
 \end{enumerate}
 
-\bibliographystyle{plain}
-\bibliography{sheet4}
+
+
+
+\begin{enumerate}
+
+\item It is not possible, since we cannot obtain $Q^c$ (or $\mathcal L^\downarrow(Q^c)$) just from $Q$, $\pi$ and $c$. \newline For example, take $q = 1$, $Q$ any 1-polytope (any interval), $\pi(x,y) = x$ and $c = (0,1)$, then $\pi^c$ is the identity in $P$, so $Q^c = P$, but we are not given $P$, so it's impossible to get $Q^c$ or just $\mathcal L^\downarrow(Q^c)$.
+
+\item Directly from the linearity of $f$ and the linearity property of the integral:
+
+\begin{equation*}
+	\vol(P) f(p_0)
+	= \vol(P) f\left( \frac1{\vol (P)}\int_P x\,\mathrm{d}x \right)
+	= f\left(\int_P x\,\mathrm{d}x \right) 
+	= \int_P f(x) \,\mathrm{d}x
+\end{equation*}
+
+\item I will enumerate the parts of the sketch of Ziegler's proof which are not clear and I'll explain them:
+
+\begin{enumerate}[(i)]
+
+\item $\Sigma(P,Q)$ is convex.
+
+\begin{proof}
+Given two points $a,b \in \Sigma(P,Q)$ and their corresponding sections $\gamma_a, \gamma_b$. Then $(ta + (1-t)b) = \int t\gamma_a + (1-t)\gamma_b = \int \gamma dx$ since the convex combination of 2 sections is a section (by the linearity of the integral and $\pi$).
```

Comment:
The first sentence lacks a verb!

Reply:
Replace "2 sections" by "two sections", see [Knuth, Mathematical Writing, p111/119](https://jmlr.csail.mit.edu/reviewing-papers/knuth_mathematical_writing.pdf)